### ISSUE TITLE
Crash in getTextAlignment with Arabic locale

### ISSUE THREAD

**Issue by SUPERCILEX** [sjwall/MaterialTapTargetPrompt#64]
I'm seeing this crash in Firebase with the 1.9.4 release:
```
Caused by java.lang.IllegalArgumentException:
java.text.Bidi.<init> (Bidi.java:215)
java.text.Bidi.<init> (Bidi.java:260)
uk.co.samuelwall.materialtaptargetprompt.MaterialTapTargetPrompt$Builder.getTextAlignment (MaterialTapTargetPrompt.java:2285)
uk.co.samuelwall.materialtaptargetprompt.MaterialTapTargetPrompt$Builder.create (MaterialTapTargetPrompt.java:2129)
uk.co.samuelwall.materialtaptargetprompt.MaterialTapTargetPrompt$Builder.show (MaterialTapTargetPrompt.java:2165)
com.supercilex.robotscouter.ui.teamlist.TutorialHelper.showCreateFirstTeamPrompt (TutorialHelper.java:40)
com.supercilex.robotscouter.ui.teamlist.TeamListActivity.onCreate (TeamListActivity.java:54)
android.app.Activity.performCreate (Activity.java:5231)
android.app.Instrumentation.callActivityOnCreate (Instrumentation.java:1087)
android.app.ActivityThread.performLaunchActivity (ActivityThread.java:2148)
android.app.ActivityThread.handleLaunchActivity (ActivityThread.java:2233)
android.app.ActivityThread.access$800 (ActivityThread.java:135)
android.app.ActivityThread$H.handleMessage (ActivityThread.java:1196)
android.os.Handler.dispatchMessage (Handler.java:102)
android.os.Looper.loop (Looper.java:136)
android.app.ActivityThread.main (ActivityThread.java:5001)
java.lang.reflect.Method.invokeNative (Method.java)
java.lang.reflect.Method.invoke (Method.java:515)
com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run (ZygoteInit.java:785)
com.android.internal.os.ZygoteInit.main (ZygoteInit.java:601)
dalvik.system.NativeStart.main (NativeStart.java)
```

#### Here's some extra info if it can help:

**Device**
Manufacturer: LGE
Model: Nexus 5
Board: Hammerhead
Android API: 19
Android OS: 4.4.4
Locale: ar

**Comment by sjwall** [comment]
Looks like the issue is related to the secondary text being null. I have fixed this and is released in v1.9.5. Reopen this issue if it doesn't, thanks.

**Comment by SUPERCILEX** [comment]
@sjwall Awesome!
